industry_benchmarks/utils/extras/plot_results.py

```diff
@@ -0,0 +1,295 @@
+    """
+    # define a ground data
+    ground = ReferenceState()
+
+    fe_results = {'Experimental': {}, 'Calculated': []}
+
+    # Lead the Measurements
+    for entry in exp_data:
+        m = Measurement(
+            labelA=ground,
+            labelB=entry,
+            DG=exp_data[entry]['exp_dG'] * unit.kilocalorie_per_mole,
+            uncertainty=exp_data[entry]['exp_dG_err'] * unit.kilocalorie_per_mole,
+            computational=False
+        )
+        fe_results['Experimental'][m.labelB] = m
+
+    for entry in calc_data:
+        m = Measurement(
+            labelA=calc_data[entry]['ligand_i'],
+            labelB=calc_data[entry]['ligand_j'],
+            DG=calc_data[entry]['dG'] * unit.kilocalorie_per_mole,
+            uncertainty=calc_data[entry]['dG_err'] * unit.kilocalorie_per_mole,
+            computational=True
+        )
+        fe_results['Calculated'].append(m)
+    
+    # Feed into the FEMap object
+    femap = FEMap()
+
+    for entry in fe_results['Experimental'].values():
+        femap.add_measurement(entry)
+
+    for entry in fe_results['Calculated']:
+        femap.add_measurement(entry)
+
+    femap.generate_absolute_values()
+
+    return femap
+
+
+def plot_femap(
+    femap: FEMap,
+    exp_data: dict[str, dict[str, float]],
+    ddg_filename : str,
+    dg_filename : str
+ ) -> None:
+    """
+    Helper method to plot out ddG and dG plots.
+    
+    Parameters
+    ----------
+    femap : FEMap
+      The cinnabar FEMap to plot
+    exp_data : dict[str, dict[str, float]]
+      Experimental data to use for shifting the absolute values
+    ddg_filename : str
+      The name of the ddg plot file.
+    dg_filename : str
+      The name of the dg plot file.
+    """
+    cinnabar_plotting.plot_DDGs(
+        femap.to_legacy_graph(),
+        figsize=5,
+        filename=ddg_filename,
+        xy_lim=[-5, 5],
+    )
+    
+    shift = sum([i['exp_dG'] for i in exp_data.values()]) / len(exp_data)
+    
+    cinnabar_plotting.plot_DGs(
+        femap.to_legacy_graph(),
+        figsize=5,
+        shift=shift,
+        filename=dg_filename,
+        xy_lim=[-15, -5],
+    )
+    
+
+def plot_schrodinger_comparison(
+    exp_data: dict[str, dict[str, float]],
+    filename: str,
+) -> None:
+    """
+    Helper method to plot out a dG comparison between experimental,
+    and schrodinger.
+    
+    Parameters
+    ----------
+    exp_data: dict[str, dict[str, float]]
+      The data loaded from the exp data file.
+    filename : str
+      The name of the plot file.
+    """
+    exp = []
+    exp_err = []
+    fep = []
+    fep_err = []
+    for entry in exp_data:
+        exp.append(exp_data[entry]['exp_dG'])
+        exp_err.append(exp_data[entry]['exp_dG_err'])
+        fep.append(exp_data[entry]['fep_dG'])
+        fep_err.append(exp_data[entry]['fep_dG_err'])
+    
+    shift = sum([i['exp_dG'] for i in exp_data.values()]) / len(exp_data)
+    
+    cinnabar_plotting._master_plot(
```

Comment:
For some reason the DG Schrodinger vs. Exp. plot says 'DDG' on the axes, do you see the same? Could adding a `plot_type='dg'` or something like that help?

Reply:
looks like `quantity` is what was missing, thanks!